source/2020-05-15-2020-ember-roadmap.md

```diff
@@ -0,0 +1,101 @@
+3. Discussion on [https://discuss.emberjs.com](https://discuss.emberjs.com/), Discord, Twitter, and other public venues (see the [community page](https://emberjs.com/community/) for how to access these communnities).
+4. Deliberations among the Ember core teams.
+5. Your comments and feedback on this document via [the RFC process](https://github.com/emberjs/rfcs/pull/519).
+
+The goal of the roadmap is to align the Ember community around a set of shared, achievable goals that balance the needs of existing users with the need to grow and support new users.
+
+## Our Roadmap
+
+Now that Ember Octane has shipped, it’s time to turn our attention to new efforts in 2020. Our goal is to build on Octane's release and capitalize on that cutting-edge foundation.
+
+- **Invest in Octane.** Octane's mental model and basic components are complete, but a number of practical and conceptual gaps remain. We will close these gaps with work on tooling, by deprecating classic APIs to simplify Ember for new users, and by introducing additional functionality where appropriate.
+- **Modernize our build system.** This year we will prioritize improvements to the Ember application build pipeline, and to Ember itself, which will bring modern optimizations like tree shaking and code splitting to both new applications *and* existing codebases.
+- **Better a11y by default**. We will better support assistive technologies via updates to the router. Additionally we will provide developers more tools for understanding and improving the accessibility of their Ember applications. Our goal is a great "out of the box" experience with Ember and assistive technologies.
+- **Share Octane outside our community.** Octane's release put Ember in front of a lot of new eyes. We will continue that trend through 2020 by talking about Octane in front of new audiences.
+
+### Invest in Octane
+
+Ember Octane put the framework on a strong footing by modernizing its most foundational APIs. Many teams are already productive using Octane, and from their experience have provided a torrent of real-world feedback. We will continue improving the developer experience (DX) of Octane throughout 2020.
+
+Many of the rough edges in Octane aren't on the features themselves, but in the supporting tooling. The usefulness of stack traces from Glimmer, the ability to use TypeScript with Ember templates, how tracked properties and Glimmer components are reflected in the Ember Inspector, and the build speed of our application pipelines are all important parts of Octane's DX. We will invest in these areas of work.
+
+For developers who are new to Ember, the presence of classic non-Octane APIs can be disorienting. We will look for creative solutions that make those features trivial for existing apps to continue using while also making them less expensive (in payload, performance, and mental model) for new adopters.
+
```

Comment:
```suggestion
<!--alex enable nuts-->
```

same comment as above, sometimes this needs a line break around it, check the preview first

re: https://github.com/ember-learn/ember-blog#contributing-troubleshooting